assets/src/styles/campaigns/themes/_theme_antarctic.scss

```diff
@@ -61,15 +61,17 @@ body.theme-antarctic {
   .page-header {
     @include page-header-block($sanctuary);
 
+    .container > *:not(.row) {
```

Comment:
Why do you we need these in campaigns only?

Reply:
The change in font size is significant in x-large-and-up, and was breaking the previous layout of many titles I saw, so I tried to keep them more or less with the same number of break lines. But I understand if you prefer to drop specific rules, that also makes sense

Reply:
Makes sense to provide continuity for existing content :+1: Can you give an example of a page where it breaks the layout?

Reply:
If we're adding the exact same rule to each campaign, we can add it in one place somewhere by using `.single-campaign` in the selector.

Reply:
Couldn't find those cases anymore (I should take more notes :sweat:), so I removed those changes; it will pop up at some point if something looks odd in some campaigns, and we can devise a proper fix at that time.